pages/api/program/[id].ts

```diff
@@ -14,69 +17,68 @@ export default async function handle(
     req: NextApiRequest,
     res: NextApiResponse,
 ): Promise<void> {
-    //TODO: Replace if statements with switch/case statements
-    if (req.method == "GET") {
-        // Obtain program id
-        const { id } = req.query;
-
-        // obtain program with provided programId
-        const program = await getProgram(id as string);
-
-        if (!program) {
-            ResponseUtil.returnNotFound(
-                res,
-                `Program with id ${id} not found.`,
-            );
-            return;
+    switch (req.method) {
+        case "GET": {
+            const { id } = req.query;
+            const result = await getProgramCardInfo(id as string);
```

Comment:
Let's convert to a number and pass it in as a number instead.

See below on how to convert to a number here. Perhaps we could make this an util function? :O 

https://github.com/uwblueprint/social-diversity-for-children/blob/862fabc0e0dd0709180aea47a5ddb30ae6e187d7/pages/api/enroll/child.ts#L43-L63